doc/doxygen/src/Glossary.dox

```diff
@@ -382,6 +382,12 @@
  * returned @ref entity_reference to determine the manager's chosen
  * location for the data and use this for any write operations.
  *
+ * As much relevant available information as possible should be provided
+ * ahead of time to `preflight`, so that the manager can prepare the
+ * backend and provide the most appropriate responses to subsequent
+ * `resolve` and `register` requests, or error early if publishing is
+ * likely to fail.
```

Comment:
[Consider] Same thing here. Is "All available information" what we want to be saying?

Reply:
Same re: Its the data owned by the host vs stuff it wants to query.

Reply:
Reworded and expanded the glossary entry in daa8a1ac.  Does this get the heart of the difference between various sources of information during the publishing process?

Reply:
Reworded yet again in e9b9f191, based on offline discussion.

Reply:
LGTM 👍

Reply:
One last tweak, mostly adding more cross-references: d691db4d